scot/datatools.py

```diff
@@ -153,6 +155,16 @@ def randomize_phase(data):
         plt.show()
     """
     data = np.asarray(data)
-    data_freq = np.fft.rfft(data, axis=0)
+    data_freq = np.fft.rfft(data)
     data_freq = np.abs(data_freq) * np.exp(1j*np.random.random_sample(data_freq.shape)*2*np.pi)
-    return np.fft.irfft(data_freq, data.shape[0], axis=0)
\ No newline at end of file
+    return np.fft.irfft(data_freq, data.shape[-1])
+
+
+def atleast_3d(x):
+    x = np.asarray(x)
+    if x.ndim >= 3:
+        return x
+    elif x.ndim == 2:
+        return x[np.newaxis, ...]
```

Comment:
The ellipsis object `...` (don't ask me how this is valid Python syntax) automatically expands to all axes that have not explicitly specified. So, in our case this should be equivalent to `x[np.newaxis, :, :]`. Can you test this and maybe replace it (explicit is better than implicit)?